src/services/fileStorage/proxy-service.js

```diff
@@ -165,46 +170,52 @@ const fileStorageService = {
 			.then((allowedFiles) => {
 				const files = allowedFiles.filter(f => f);
 				if (!files.length) {
-					return new Forbidden();
+					return Promise.reject(new Forbidden());
```

Comment:
Sollte doch okay sein die Exception zu werfen, wenn im Catch eh rejected wird, oder?

Reply:
Also meines Wissens nach wirft `new Forbidden()` keine Exception in dem Sinne. D.h. das der HTTP-Status-Code nicht 403 ist. Das hat nur funktioniert, wenn man gleichzeitig eine Rejection zurück gibt. Kann mich auch irren.

Reply:
jup. in promise.reject sollte eine exception rein